src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/metricsdb/MetricsDB.java

```diff
@@ -183,6 +184,17 @@ public void putMetric(Metric<Double> metric, Dimensions dimensions, long windowS
         .execute();
   }
 
+  /**
+   * Drop a metric table. This is for IT framework to use only
+   * @param metricName metric table to be deleted
+   */
+  @VisibleForTesting
```

Comment:
Do we need `VisibleForTesting` annotation for public method?

Reply:
this function needs to be called outside of MetricDB but is intended to be used by UT only. The metricDB is written only once in prod.